src/tools/JitOrderParser/JitOrderParser.csproj

```diff
@@ -0,0 +1,8 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
```

Comment:
I think at this point we should target netcoreapp3.0

Reply:
OK